.github/workflows/ci.yaml

```diff
@@ -112,52 +112,6 @@ jobs:
         with:
           arguments: ':schema-loader:dockerfileLint'
 
-  build-check-example-project:
-    name: Build check for 'Getting Started' example project
-    runs-on: ubuntu-latest
-
-    defaults:
-      run:
-        working-directory: docs/getting-started
-
-    steps:
-      - uses: actions/checkout@v4
-
-      - name: Set up JDK ${{ env.JAVA_VERSION }} (${{ env.JAVA_VENDOR }})
-        uses: actions/setup-java@v4
-        with:
-          java-version: ${{ env.JAVA_VERSION }}
-          distribution: ${{ env.JAVA_VENDOR }}
-
-      - name: Setup Gradle
-        uses: gradle/actions/setup-gradle@v3
-
-      - name: Build Getting Started project
-        run: ./gradlew assemble
-
-  build-check-example-project-for-kotlin:
```

Comment:
As I commented on https://github.com/scalar-labs/scalardb-samples/pull/68#issuecomment-2259485312, this isn't copied to `scalardb-samples`. Probably it would be great if we add equivalent build tests to `scalardb-samples` later.

Reply:
@komamitsu Yes. We should do that. Thanks. 

@KodaiD Let's do that later.